Vungle/src/main/java/com/applovin/mediation/adapters/VungleMediationAdapter.java

```diff
@@ -354,24 +354,10 @@ private void showAdViewAd(final boolean isBiddingAd, final String placementId, f
     {
         log( "Showing " + ( isBiddingAd ? "bidding " : "" ) + adFormatLabel + " ad for placement: " + placementId + "..." );
 
-        if (bannerAd != null) {
-            Context context = getContext( null );
-            RelativeLayout layout = new RelativeLayout( context ) {
-                @Override
-                protected void onAttachedToWindow() {
-                    super.onAttachedToWindow();
-                    BannerView bannerView = bannerAd.getBannerView();
-                    if (bannerView != null) {
-                        if (bannerView.getParent() != null) {
-                            ((ViewGroup) bannerView.getParent()).removeView( bannerView );
-                        }
-                        bannerView.setGravity( Gravity.CENTER );
-                        addView( bannerView );
-                    }
-                }
-            };
-
-            listener.onAdViewAdLoaded( layout );
```

Comment:
I recall this was very intentional fix, we don't need it anymore?

Reply:
Yes, we fixed it from 7.1.0-QA3 build.